arctic/tickstore/tickstore.py

```diff
@@ -636,6 +636,12 @@ def _ensure_supported_dtypes(array):
             array = array.astype('<f8')
         elif (array.dtype.kind) in ('U', 'S'):
             array = array.astype(np.unicode_)
+        elif (array.dtype.kind) == 'O':
+            try:
+                array = np.array([s.encode('ascii') for s in array])
```

Comment:
Is real unicode support available?  Would it be possible to use UTF-8?  Not essential but might as well have it given that it is easy to slip into UTF8 sometimes.

Reply:
yeah thats easy enough to change